docs/deployment/managed-scanning/bitbucket.md

```diff
@@ -58,9 +58,11 @@ See [Pre-deployment checklist > Permissions](/deployment/checklist#permissions)
 1. Repeat the steps above for each additional Bitbucket workspace you'd like added to Semgrep.
 <!-- vale on -->
 
+<!-- For Katie: How does this add a repository? I followed the instructions up to **Set up Managed Scans** but did I miss the part where the user selects repositories? Or does this add ALL repositories? -->
```

Comment:
For @katie to confirm

Reply:
Yes, once you add the workspace, Semgrep pulls in all the repos in that workspace (you can see in this [demo by Andy at the end](https://www.loom.com/share/9b85bd9938474afcbf571c3f700674f0) that once he adds the workspace, multiple repos start scanning). So this answers your other question to, that SMS performs a full scan in batches on all the repos _in that workspace_